source/getting-started/index.md

```diff
@@ -65,10 +81,12 @@ npm install -g phantomjs2
 If you run into any problems with this step, consult the [PhantomJS download
 instructions](http://phantomjs.org/download.html).
 
-To verify that your installation was successful, run:
+Once we have everything installed, we'll be able to run commands with Ember CLI's binary, which is `ember`. There are many commands available by running the binary `ember` along with another command. A list of all commands is available by simply running `ember` or `ember help` from the command line.
```

Comment:
I don't think `ember` is technically binary... maybe "executable" is a better word?